server/api_geo_stats.go

```diff
@@ -260,19 +191,26 @@ type MOERData struct {
 }
 
 // loadMOERData loads MOER data from CSV files for the specified balancing authority.
-func loadMOERData(ba string) (map[string]map[string][]float64, error) {
-	folderPath := filepath.Join("watttime_data", ba)
+func (gcas *GCAServer) loadMOERData(ba string) (map[string]map[string][]float64, error) {
+	dataPath := filepath.Join(gcas.baseDir, "watttime_data", ba)
```

Comment:
it's probably worth hardcoding 2023 here so we don't run into issues when we update to 2024.

And really, probably worth adding a const somewhere and removing all the hardcoded instances of 2023